Dockerfile.add-frontend

```diff
@@ -1,7 +1,6 @@
 FROM gcr.io/spiffe-io/spire-server:1.1.3
 RUN apk add curl
-COPY bin/tornjak-agent tornjak-agent
-COPY ui-agent ui-agent
+COPY bin/tornjak-backend tornjak-backend
```

Comment:
Should this file be called `Dockerfile.add-backend`?

Reply:
good catch! renamed